standard_schema/hedwiki/HED8.2.0.mediawiki

```diff
@@ -196,41 +196,47 @@ Each term in this vocabulary has a human-readable description and may include ad
-***** Thigh <nowiki>[Upper part of the leg between hip and knee.]</nowiki>
+******* Teeth <nowiki>[The hard bonelike structures in the jaws. A collection of teeth arranged in some pattern in the mouth or other part of the body.]</nowiki>
+**** Neck <nowiki>[The part of the body connecting the head to the torso, containing the cervical spine and vital pathways of nerves, blood vessels, and the airway.]</nowiki>
 **** Torso <nowiki>[The body excluding the head and neck and limbs.]</nowiki>
-***** Torso-back <nowiki>[The rear surface of the human body from the shoulders to the hips.]</nowiki>
-***** Buttocks <nowiki>[The round fleshy parts that form the lower rear area of a human trunk.]</nowiki>
 ***** Torso-chest <nowiki>[The anterior side of the thorax from the neck to the abdomen.]</nowiki>
-***** Gentalia <nowiki>{deprecatedFrom=8.1.0}[The external organs of reproduction.]</nowiki>
-***** Hip <nowiki>[The lateral prominence of the pelvis from the waist to the thigh.]</nowiki>
-***** Waist <nowiki>[The abdominal circumference at the navel.]</nowiki>
+***** Torso-back <nowiki>[The rear surface of the human body from the shoulders to the hips, often regarded as the back]</nowiki>
+***** Abdomen <nowiki>[The part of the body between the thorax and the pelvis.]</nowiki>
+***** Pelvis <nowiki>[The bony structure at the base of the spine supporting the legs.]</nowiki>
+****** Buttocks <nowiki>[The round fleshy parts that form the lower rear area of a human trunk.]</nowiki>
+****** Hip <nowiki>[The lateral prominence of the pelvis from the waist to the thigh.]</nowiki>
+****** Waist <nowiki>[The abdominal circumference at the navel.]</nowiki>
+****** Gentalia <nowiki>{deprecatedFrom=8.1.0}[The external organs of reproduction.]</nowiki>
+****** Genitalia <nowiki>[The external organs of reproduction and urination, located in the pelvic region. This includes both male and female genital structures.]</nowiki>
 **** Upper-extremity <nowiki>[Refers to the whole superior limb (shoulder, arm, elbow, wrist, hand).]</nowiki>
-***** Elbow <nowiki>[A type of hinge joint located between the forearm and upper arm.]</nowiki>
+***** Upper-arm <nowiki>[Portion of arm between shoulder and elbow.]</nowiki>
+****** Shoulder <nowiki>[Joint attaching upper arm to trunk.]</nowiki>
 ***** Forearm <nowiki>[Lower part of the arm between the elbow and wrist.]</nowiki>
+****** Elbow <nowiki>[A type of hinge joint located between the forearm and upper arm.]</nowiki>
 ***** Hand <nowiki>[The distal portion of the upper extremity. It consists of the carpus, metacarpus, and digits.]</nowiki>
+***** Wrist <nowiki>[A joint between the distal end of the radius and the proximal row of carpal bones.]</nowiki>
+****** Palm <nowiki>[The part of the inner surface of the hand that extends from the wrist to the bases of the fingers.]</nowiki>
+****** Knuckles <nowiki>[A part of a finger at a joint where the bone is near the surface, especially where the finger joins the hand.]</nowiki>

```

Comment:
Same thing here with adding parts.  We need to discuss...

Reply:
Sorry, the Wrist should be under Hand.